### DISCUSSION TITLE
Wrong height for Label placed inside a Scrollview

### DISCUSSION THREAD

**Discussion by cat0363** [dotnet/maui#13494] [Q&A]
Xamarin.Forms and .NET MAUI have different heights when placing a Label inside a ScrollView.
Below is the XAML for confirmation. (This is the result of running on Android.)

    <Grid RowDefinitions="44,*" RowSpacing="0">
      <ScrollView Grid.Row="0" Orientation="Horizontal" BackgroundColor="Red">
        <Label Text="This is test." FontSize="18" HorizontalOptions="Start" VerticalOptions="Center" BackgroundColor="Yellow" />
      </ScrollView>
      <Grid Grid.Row="1" BackgroundColor="Blue" />
    </Grid>

The execution result of Xamarin.Forms is as follows.
![image](https://user-images.githubusercontent.com/125236133/220499364-1890ae45-6207-4126-9683-fddc87dd70c3.png)

The execution result of .NET MAUI is as follows.
![image](https://user-images.githubusercontent.com/125236133/220499416-1c998220-6d04-4d24-93a4-f2fcc9e34b42.png)

How can I get the same execution result as Xamarin.Forms?

**Comment by acaliaro** [comment]
If you change 44 to Auto?

**Reply by cat0363** [reply]
@acaliaro, thank you for your reply.
It's working as expected.

    <Grid RowDefinitions="Auto,*" RowSpacing="0">
      <ScrollView Grid.Row="0" Orientation="Horizontal" BackgroundColor="Red" HeightRequest="44">
        <Label Text="This is test." FontSize="18" HorizontalOptions="Start" VerticalOptions="Center" BackgroundColor="Yellow" />
      </ScrollView>
      <Grid Grid.Row="1" BackgroundColor="Blue" />
    </Grid>

After changing the Grid's RowDefinition from 44 to Auto and adding HeightRequest="44" to the 
ScrollView, it's now the same as Xamarin.Forms.

Is it normal behavior in .NET MAUI that the height of the label is not what you expect if you explicitly 
specify the height in the RowDefinition?

**Reply by acaliaro** [reply]
I don't know, maybe you could open an issue with a repo. I know that in MAUI some values regarding spacing, margin, padding are changed from XF

**Reply by cat0363** [reply]
@acaliaro , thank you very much for your help. 
I would like to report it as an issue.
